### DISCUSSION TITLE
jwt authentication token

### DISCUSSION THREAD

**Discussion by behdad088** [dotnet-architecture/eShopOnContainers#2116] [General]
Hi, I have a question related to authentication in this ApiGateways. In ApiGateways we use jwt token to create token for mobileshoppingagg and webshoppingaggswaggerui and with the help of HttpClientAuthorizationDelegatingHandler we use the same token to call Order API. My question here is that how does a token that was created with Audience mobileshoppingagg or webshoppingaggswaggerui works for the order api? 
Is there another setting that I am missing?

**Comment by PeterSHL** [comment]
If you have a look at [CommonExtensions.cs](https://github.com/dotnet-architecture/eShopOnContainers/blob/06d5164532902a076813894c965d6c99122d39f2/src/Services/Services.Common/CommonExtensions.cs) you can see that options.TokenValidationParameters.ValidateAudience is set to false.
It looks like the Order API doesn't validate the audience.
